100_Numpy_exercises_with_solutions.md

```diff
@@ -682,7 +682,7 @@ print(F)
 
 w,h = 16,16
 I = np.random.randint(0,2,(h,w,3)).astype(np.ubyte)
-F = I[...,0]*256*256 + I[...,1]*256 +I[...,2]
+F = I[...,0]*(256*256) + I[...,1]*256 +I[...,2]
```

Comment:
Thanks. Maybe you can also add a comment with the previous faulty line explaining it is wrong because of type promotion.

Reply:
Sorry, I meant leaving a comment in the solution explaining the difference between the right and faulty version.
Something like:
```
# Parenthesis around 256*256 are necessary to ensure proper dtype promotion else you can overflow
```

Reply:
Sorry, I am not familiar with how to make a PR properly. Thanks for guide.